src/plugins/discover/public/application/main/utils/get_fetch_observable.ts

```diff
@@ -53,7 +53,6 @@ export function getFetch$({
         );
       })
     ),
-    data.query.queryString.getUpdates$(),
```

Comment:
logic triggering a fetch when the query changes migrated to [build_state_subscribe.ts](https://github.com/elastic/kibana/pull/159175/files#diff-bd94040de177c0274828185d6ca832036cbb79ee8b796d8012cb7e69c0dd21c7)

Reply:
is is due to the fact that we need to create an ad-hoc data view for text-based languages before starting a fetch